Cargo.toml

```diff
@@ -19,5 +19,5 @@ git-fetch-with-cli = true
 
 [package]
 name = "daft"
-version = "0.0.20"
+version = "0.0.0"
```

Comment:
It seems this is always patched before we build and publish a new Daft release - @samster25  is this a safe way of determining whether this is a Daft release build vs a local/dev build? This PR currently uses Daft version `0.0.0` as a corner-case to disable analytics.

Reply:
I wouldn't go about it that way, it would be better to build a flag for a release build rather than relying on the version number.